lib/generators/spotlight/install_generator.rb

```diff
@@ -1,3 +1,4 @@
+# rubocop:disable ClassLength
```

Comment:
Ah! I like that, I actually didn't know that syntax was possible, but makes sense!